src/index.js

```diff
@@ -29,97 +27,137 @@ function handleConsentResult(tcfApi, store = {}, vendorList, consentData = {}) {
 		policyVersion: consentPolicyVersion = 1
 	} = consentData;
 
-	const tcfApiCall = (command) => {
-		tcfApi(command, 2, () => log.debug(`${command} command was called`));
-	};
+	let displayOptions = {};
+
+	if (!config.autoDisplay) {
+		return { display: false };
+	}
 
 	if (!created) {
 		log.debug('No consent data found. Showing consent tool');
-		config.autoDisplay && tcfApiCall('showConsentTool');
+		displayOptions = {display: true, fn: 'showConsentTool'};
```

Comment:
I think that `command` instead `fn` would be more descriptive

Reply:
done